mq/integrations/splunk/index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Splunk Modular Input
+layout: default
+section: mq
+breadcrumbs:
+- ['MQ Integrations', '/mq']
+- ['Splunk Modular Input', '/splunk']
+---
+
+<section id="toc">
+  <h3>Table of Contents</h3>
+  <ul>
+    <li><a href="#download">Download IronMQ Modular Input</a></li>
+    <li><a href="#install">Installing IronMQ into Splunk</a></li>
+    <li><a href="#data">Adding IronMQ as a Data Source in Splunk</a></li>
+    <li><a href="#use">Using IronMQ with Splunk</a></li>
+    <li><a href="#reference">Additional Reference</a></li>
+  </ul>
+</section>
+
+[Splunk](http://www.splunk.com) is the leading platform for Operational Intelligence by providing insight into a wide range of machine data.
+
+Splunk provides an add-on environment for building integrations that are available through [Splunkbase](https://splunkbase.splunk.com/).
+
+Iron.io has built an official IronMQ Modular Input plugin for Splunk users to reliably deliver and buffer data into their Splunk environment. You must have both an Iron.io account and Splunk Enterprise account to use this integration.
+
+<h2 id="download">Download IronMQ Modular Input</h2>
+
+Go to the IronMQ Modular Input on [Splunkbase](https://splunkbase.splunk.com/app/2744/) and download the add-on by selecting the 'Download' button. Make sure IronMQ is compatible with your version of Splunk Enterprise. 6+.
```

Comment:
That's okay, we want users to use our libraries through us directly. I'll make a note in the Docs.